doc/7/controllers/security/m-get-users/index.md

```diff
@@ -0,0 +1,41 @@
+---
+code: true
+type: page
+title: mGetUsers
+description: Gets multiple security users
+---
+
+# mGetUsers
+
+Gets multiple security users.
+
+<br />
+
+```js
+mGetUsers(kuids, [options]);
+```
+The route exists in `GET` and `POST`.
+By default, the SDK hits the `GET` one.
+You can force it to be `POST` in the `options`.
+
+<br />
+
+| Property | Type | Description |
+|--- |--- |--- |
+| `ids` | <pre>array&lt;string&gt;</pre> | User identifiers |
```

Comment:
our standard is more like this:

```suggestion
| `ids` | <pre>string[]</pre> | User identifiers |
```

(see [this route](https://docs.kuzzle.io/sdk/js/7/controllers/document/m-create/) for instance)